Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2381,6 +2381,166 @@ def _is_status_code_valid(self, response, ok_codes=None):
                 return response.status_code in status_codes
             return response.ok
 
+    class SessionBaseClient(BaseClient):
```

Comment:
why do we need 2 classes? BaseClient and SessionBaseClient?
Cant SessionBaseClient totally replace BaseClient?
Are there cases where SessionBaseClient will behave differently?

Reply:
@jochman What do you say? Would BaseClient act the same if we just add session to it?

Reply:
After checking it out, changing the `BaseClient` to a session will not break the unit tests.
@orlichter1 guided to open a new branch and change the `requests.request` to `session`. Then we will check if it breaks tests.